packages/kit/src/cli.js

```diff
@@ -40,13 +41,25 @@ prog
 	.describe('Synchronise generated files')
 	.option('--mode', 'Specify a mode for loading environment variables', 'development')
 	.action(async ({ mode }) => {
-		if (!fs.existsSync('svelte.config.js')) {
-			console.warn('Missing svelte.config.js — skipping');
+		const event = process.env.npm_lifecycle_event;
+
+		// TODO remove for 1.0
+		if (event === 'prepare') {
```

Comment:
This is also a lot cleaner than reading the package.json and testing the prepare script ourselves.
However it is going to cause a false positive if users have added their own prepare scripts i think.

E.g. husky the popular git hooks automation uses it in their installations: https://typicode.github.io/husky/#/?id=install

so a test if it actually is our own prepare script is still needed i fear. (adding it here to only read the file if we are in prepare is a better way still)